packages/elements-core/src/components/Docs/HttpOperation/Request.tsx

```diff
@@ -88,36 +92,49 @@ Request.displayName = 'HttpOperation.Request';
 
 const schemeExpandedState = atomWithStorage<Record<string, boolean>>('HttpOperation_security_expanded', {});
 
-const SecurityPanel: React.FC<{ scheme: HttpSecurityScheme; includeKey: boolean }> = ({ scheme, includeKey }) => {
+const SecurityPanel: React.FC<{ schemes: HttpSecurityScheme[]; includeKey: boolean }> = ({ schemes, includeKey }) => {
   const [expandedState, setExpanded] = useAtom(schemeExpandedState);
+  const collection = schemes.length > 1;
 
   return (
     <SubSectionPanel
-      title={`Security: ${getReadableSecurityName(scheme, includeKey)}`}
-      defaultIsOpen={!!expandedState[scheme.key]}
-      onChange={isOpen => setExpanded({ ...expandedState, [scheme.key]: isOpen })}
+      title={`Security: ${getReadableSecurityNames(schemes, includeKey)}`}
+      defaultIsOpen={!!expandedState[getReadableSecurityNames(schemes)]}
+      onChange={isOpen => setExpanded({ ...expandedState, [getReadableSecurityNames(schemes)]: isOpen })}
     >
-      <MarkdownViewer
-        style={{ fontSize: 12 }}
-        markdown={`${scheme.description || ''}\n\n` + getDefaultDescription(scheme)}
-      />
+      <Box m={-2}>
+        {schemes.map(scheme => (
+          <Box key={scheme.key} p={2} m={2} border>
+            {collection && (
+              <MarkdownViewer
+                style={{ fontWeight: 'bold', fontSize: 12, marginBottom: 10 }}
+                markdown={getReadableSecurityName(scheme, shouldIncludeKey(schemes, scheme.type))}
+              />
+            )}
+            <MarkdownViewer
+              style={{ fontSize: 12 }}
+              markdown={`${scheme.description ?? ''}\n\n` + getDefaultDescription(scheme)}
+            />
+          </Box>
+        ))}
+      </Box>
     </SubSectionPanel>
   );
 };
 
-const SecuritySchemes = ({ schemes }: { schemes: HttpSecurityScheme[] }) => {
+const SecuritySchemes = ({ schemes }: { schemes: HttpSecurityScheme[][] }) => {
   const { nodeHasChanged } = useOptionsCtx();
 
   if (!schemes.length) {
     return null;
   }
-
+  /* Note: not sure what to do about the nodeId below and how important it was to be tied to the scheme id like that... */
   return (
     <VStack spacing={3}>
       {schemes.map((scheme, i) => (
-        <Box pos="relative" key={i}>
-          <SecurityPanel scheme={scheme} includeKey={shouldIncludeKey(schemes, scheme.type)} />
-          <NodeAnnotation change={nodeHasChanged?.({ nodeId: scheme.id })} />
+        <Box pos="relative" key={i} p={0} data-test="security-row">
+          <SecurityPanel schemes={scheme} includeKey={shouldAddKey(scheme, schemes)} />
+          <NodeAnnotation change={nodeHasChanged?.({ nodeId: scheme.map(node => node.id).join('') })} />
```

Comment:
That will break Proposals. We'd need to pass an id of http_security_group node here.

Reply:
Moved it over to match individual schemes again instead of the security group - I *think* this will retain current functionality - take a look :)